src/index.html

```diff
@@ -11,6 +11,12 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="submit__big" data-qa="big">
+      <input value="Try &quot;Los Angeles&quot;" data-qa="keypress">
```

Comment:
Please use placeholder attr instead of value
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/54936044/9f3aed4a-0ae2-4fb9-9d05-f8858e6ea8d1)